index.html

```diff
@@ -99,15 +99,15 @@ <h3>Skills</h3><i class="fa-solid fa-angle-right"></i>
     <div class="heading-2">
       <h2>Contact me</h2>
     </div>
-    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>  
-    <form action="https://formspree.io/f/mjvzjjpb" method="post"></form>
+    <p class="contact-form-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>  
+    <form id=form action="https://formspree.io/f/mjvzjjpb" method="post">
       <div class="contact-box">
         <div class="fullname">
           <label for="name"></label>
           <input type="text" id="name" name="user_name" maxlength="30" placeholder="enter full name" required>
-        </div>
+        </div>        
         <div class="email">
-          <label for="email"></label>
+          <p class="error-message">*invalid email</p>
           <input type="email" id="mail" name="user_email" placeholder="enter email" required>
```

Comment:
- [ ] As the requirements suggest, if the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

- It shouldn't be like this:
![image](https://user-images.githubusercontent.com/52143815/203845132-adb7b68f-c9d9-4a5b-ada9-7992481261af.png)